src/main/java/com/ericsson/sm/CarApp/service/impl/ClientServiceImpl.java

```diff
@@ -62,4 +62,21 @@ public List<AllClientsResponseDto> getAll() {
         }
         return savedClients;
     }
+
+    @Override
+    public ClientResponseDto findById(Long id) {
+        Client client = clientRepository.findById(id).orElse(null);
```

Comment:
Ovo mi je ok za sada, ali kada budem radili error handling ona ćemo ovu liniju promjeniti. Ovdje je problem što se ništa ne dogodi ako ti aplikacija ne pronađe klijenta.